### DISCUSSION TITLE
Anyone interested in a gitter.im for Willow?

### DISCUSSION THREAD

**Discussion by hamishcunningham** [toverainc/willow#204] [General]
Giiter.im does a nice chat interface with github integration.
The one for the ESP32 Arduino core is very lively and useful: https://gitter.im/espressif/arduino-esp32
How about one for willow? There seem to be enough developers working with it now to make chat an efficient way to answer quick questions that don't warrant an issue or discussion?

**Comment by nikito** [comment]
I feel like this could be a good idea, or even a Discord depending on what people prefer. With a lot of new dev starting to happen as we approach 1.0 and beyond it may be a faster/more collaborative way to communicate. 😃

**Reply by stintel** [reply]
I prefer IRC. I'm not a fan of any of these hipster alternatives that eat resources like crazy and drain laptop batteries twice as fast.

**Reply by kristiankielhofner** [reply]
Unfortunately given our target user (and likely devs) - "hipsters" hahaha I think a Willow IRC channel would almost certainly be a ghost town. I myself crossed over to the hipster dark side and haven't used IRC in at least a decade.

**Comment by kristiankielhofner** [comment]
@stintel and I are more-or-less working full time on dev and likely will be even for a time post stable release so our time availability to participate in more synchronous communication is limited. We likely wouldn't be able to participate regularly and consistently. At some point in the future I hope that I personally could spend less time on dev and more time on community management/engagement but this appears to be a ways away.

With the Willow community building (like both of you!) this is less of an issue but I just want to set expectations for @stintel and I. That said we'd definitely support such a thing but I don't want the expectation set that we would be regularly available - it took me 10 days to respond to a recent Twitter DM!

**Comment by hamishcunningham** [comment]
@kristiankielhofner The arduino-esp32 chat is mostly not the core developers, though they chip in now and then. The big win is that lots of other knowledgeable people hang around, so simple questions get answered quickly without too much overhead.

Shall I set up a gitter.im then, and make you an admin @nikito? I agree with @stintel that some of the new chat systems like Discord are quite heavy, but gitter seems fairly light. And I'll do a PR to link it from the docs, and we can see if it attracts enough people to be sustainable...?

**Reply by nikito** [reply]
I'm ok with that if everyone else is. 🙂

**Comment by kristiankielhofner** [comment]
We support this and are excited for it, thanks for helping out!

**Comment by nikito** [comment]
@kristiankielhofner Is there an official Willow logo/image that you plan to use? Just asking as I think when you make a gitter.im room you can put a thumbnail image, figured if we had that we could assign it to the room. Don't think it is necessary to make the room, but figured if this already existed it could be nice to add. 😃 I think I saw some referenced in the code repos but didn't want to grab anything I am not supposed to lol.

**Comment by hamishcunningham** [comment]
Yes a logo would be great :)
Sorry to be slow, I should be able to get to this tomorrow

**Comment by hamishcunningham** [comment]
gitter.im was taken over and converted to matrix, which makes it more of a pain to get in (which works by oauth / your gitlab or github account). On the upside this might increase the signal to noise ratio :)

Perhaps try it at https://matrix.to/#/#willow-voice-assistant:gitter.im and see if you think it worth the bother?

**Reply by nikito** [reply]
Joined, seemes to work fine for me. Also have the app on my phone and that went fine too. 😃

**Comment by kristiankielhofner** [comment]
I created a new [repo](https://github.com/toverainc/willow-branding) for the Willow branding assets. Make sure to pay attention to the licensing terms but I can say there is no concern on our part with usage of these assets for this purpose.

**Reply by nikito** [reply]
@hamishcunningham Do you want to update the room when you have a moment with something from the repo above? I would do it but seems I do not have privileges to do so 😆

**Reply by kristiankielhofner** [reply]
The content is pretty out of date but the overall look/feel is more or less established:

https://heywillow.io/

**Comment by hamishcunningham** [comment]
Morning all!
@nikito and @stintel are now admins on the gitter room, and I've added the logo.
I think the best link to share is https://app.gitter.im/#/room/#willow-voice-assistant:gitter.im
Shall I do a PR to add this to WIS and Willow READMEs? And/or willow-docs? wisng branch for WIS?

**Reply by kristiankielhofner** [reply]
Good morning (I suspect afternoon for you)!

We'll be moving all docs to heywillow.io and stripping the various READMEs in repos down to more-or-less developer only documentation. They are hosted in [this](https://github.com/toverainc/willow-docs) repo and powered by mkdocs.

As willow-docs is already well behind in the coming days I will be absolutely tearing up the content to match current Willow, WIS, and WAS in addition to making it a little more of an intro/"marketing" site.

If you and/or @nikito or anyone else wants to get started on all of this with PRs, etc (I can give you write access as well) that would be very much appreciated!

**Reply by hamishcunningham** [reply]
yes, I'm up for that :)
